fury/ui.py

```diff
@@ -69,6 +69,53 @@ def clip_overflow(textblock, width, side='right'):
             return textblock.message
 
 
+def wrap_overflow(textblock, wrap_width):
+    """Wraps overflowing text of TextBlock2D with respect to width.
+
+    Parameters
+    ----------
+    textblock : TextBlock2D
+        The textblock object whose text needs to be wrapped.
+    wrap_width : int
+        Required width of the wrapped text.
+
+    Returns
+    -------
+    wrapped text : str
+        Wrapped version of the text.
+    """
+    original_str = textblock.message
+    start_ptr = 0
+    end_ptr = len(original_str)
+
+    if wrap_width <= 0:
+        raise ValueError(
+            f'Value of wrap_width cannot be less that or equal to 0'
+        )
```

Comment:
Instead of throwing an error it would be better to handle it silently. (Make the algo robust enough to handle those cases)